bot/utils/checks.py

```diff
@@ -0,0 +1,21 @@
+from typing import Optional
+
+from discord.ext.commands import CheckFailure
+
+
+class InWhitelistCheckFailure(CheckFailure):
+    """Raised when the `in_whitelist` check fails."""
+
+    def __init__(self, redirect_channel: Optional[int]) -> None:
+        self.redirect_channel = redirect_channel
+
+        if redirect_channel is not None:
+            redirect_message = (
+                f"Here, please use the <#{redirect_channel}> channel instead"
```

Comment:
```suggestion
                f" here. Please use the <#{redirect_channel}> channel instead"
```
I believe this would add capitalization and a grammatically incorrect comma in the message.